src/BoltOn.Data.CosmosDb/CosmosDbContextFactory.cs

```diff
@@ -0,0 +1,27 @@
+using System;
+
+namespace BoltOn.Data.CosmosDb
+{
+    public interface ICosmosDbContextFactory
+    {
+        TCosmosDbContext Get<TCosmosDbContext>() where TCosmosDbContext : BaseCosmosDbContext;
+    }
+
+    public class CosmosDbContextFactory : ICosmosDbContextFactory
```

Comment:
do we really need this factory? the reason why I created a factory for EF is based on IQuery, I set trackingchanges properties. As we're not doing anything like that, can't we inject the dbcontext directly?

Reply:
@gokulm : removed it .